ParkingFinder/tables/registered_vehicles.py

```diff
@@ -1,16 +1,19 @@
 from sqlalchemy import (
     Column,
     String,
+    ForeignKey,
+    PrimaryKeyConstraint,
 )
 
 from ParkingFinder.tables.base import Base
 
 
 class RegisteredVehicles(Base):
     __tablename__ = 'registered_vehicles'
+    __table_args__ = (PrimaryKeyConstraint('user_id', 'plate'), )
 
-    user_id = Column(String(64), nullable=False)
-    plate = Column(String(7), nullable=False)
+    user_id = Column(String(64), ForeignKey('users.user_id'), nullable=False)
+    plate = Column(String(7), ForeignKey('vehicles.plate'), nullable=False)
     
```

Comment:
I think index constraint for each column is enough because we would never query the table by (user_id, plate).